### PR TITLE
Added fortune methods

### PR DESCRIPTION
**Pull request by lottiely**
Added
- `show_all()`
  - Returns all fortunes from the database
- `get_fortune()`
  - Returns a random fortune
- `delete_fortune(id)`
  - Deletes a fortune from the database by ID

Issue
#2 

Related commits b5fba8df3d7eea1748882725c83ee3c620f7b202 178da46359401cd73add00529c7a8f832cc99e25

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -72,37 +77,71 @@ def test_endpoint():
...
     else:
         items = response_obj['Items']
         return items
         # testing prints
         #print("GetItem succeeded: ")
         #print(json.dumps(item, indent=4, cls=DecimalEncoder))
-    #return jsonify(response_obj) this is for Dustin
 
+def delete_fortune(id):
+    # connect to the database
```
Comment (GiantRobato) [inline src/app.py]: you can delete this since the next line is basically the same thing.

File: src/app.py [hunk 2]
```diff
@@ -72,37 +77,71 @@ def test_endpoint():
...
+            }
+        )
+    except ClientError as e:
+        if e.response['Error']['Code'] == "ConditionalCheckFailedException":
+            print(e.response['Error']['Message'])
+        else:
+            raise
+    else:
+        print("DeleteItem succeeded: ")
+        print(json.dumps(response, indent=4))
```
Comment (GiantRobato) [inline src/app.py]: these should be debug logs

File: src/app.py [hunk 3]
```diff
@@ -72,37 +77,71 @@ def test_endpoint():
...
+    
+@app.route("/fortune", methods=["GET"])
+def get_fortune():
+    fortunes = get_all_fortunes()
+
+    # Choose a random fortune
+    chosen = random.randint(0, len(fortunes)-1)
+    print(fortunes[chosen])
```
Comment (GiantRobato) [inline src/app.py]: this should be a debug log

File: src/app.py [hunk 4]
```diff
@@ -72,37 +77,71 @@ def test_endpoint():
...
+    else:
+        print("DeleteItem succeeded: ")
+        print(json.dumps(response, indent=4))
+    
+@app.route("/fortune", methods=["GET"])
+def get_fortune():
+    fortunes = get_all_fortunes()
+
+    # Choose a random fortune
+    chosen = random.randint(0, len(fortunes)-1)
```
Comment (GiantRobato) [inline src/app.py]: 2 things. 1) use `random.choice(list))` so the code is more readable which allows 2) removing of the comment

**Review by GiantRobato — CHANGES_REQUESTED** [review]
Replace prints with debugs and use `random.choice`

File: src/app.py (hunk 4 above)
Comment (lottiely) [reply src/app.py]: Updated. :)

File: src/app.py (hunk 3 above)
Comment (lottiely) [reply src/app.py]: Updated. :)

File: src/app.py (hunk 2 above)
Comment (lottiely) [reply src/app.py]: Updated. :)

File: src/app.py (hunk 1 above)
Comment (lottiely) [reply src/app.py]: Updated. :)

File: src/app.py [hunk 5]
```diff
@@ -23,86 +28,85 @@ def default(self, o):
...
+
+def createItem(fortune_item):
     dynamodb = boto3.resource('dynamodb', region_name='us-west-2')
 
-    # Access database table
     table = dynamodb.Table('FortuneCookie')
 
-    # Make JSON file of fortunes you want to add
-    # rawrawrawr stuff
     id = str(uuid.uuid4())
```
Comment (GiantRobato) [inline src/app.py]: id should never be used as an local var, use _id or id_ since `id` is a builtin method

**Review by GiantRobato — CHANGES_REQUESTED** [review]
Fix variable name

File: src/app.py (hunk 5 above)
Comment (ericpak) [reply src/app.py]: __id__

Comment (ericpak) [reply src/app.py]: _id_

Comment (lottiely) [reply src/app.py]: Thanks! I changed it.

**Review by GiantRobato — APPROVED** [review]

**Pull request merged**